src/blockdata/transaction.rs

```diff
@@ -74,19 +74,6 @@ impl OutPoint {
     }
 
     /// Checks if an `OutPoint` is "null".
-    ///
```

Comment:
I think this example should stay here. It should be changed to something which doesn't use `genesis_block()`

Reply:
In Tapyrus, coinbase transaction of a genesis block has previous outpoint.
This example is  not appropriate as a usage of is_null() mehtod.

Reply:
Oh, I missed it. I understand the situation. I'm going to merge this PR and thanks for creating PR #17. 👍